README.md

```diff
@@ -1,5 +1,4 @@
 # FCC Test Suite Template
```

Comment:
-kindly update your Readme file with yours to become descriptive of your current work.  Because others can understand your work by reading this file

Reply:
- [ ] please make sure to check for spelling and grammatical errors so as to ensure that students can fully understand what changes you are trying to request. :+1: :sparkles: